### ISSUE TITLE
Issue - demo2

### ISSUE THREAD

**Issue by GiuseppeArbore** [Zurehma/SE2-Kiruna_eXplorer#3]
You've done a great job with the user-experience.
We have a few suggestions for further improvement:

- Taller text-field by default for description
- Make the form more coherent, removing empty spaces
- Add the possibility to delete custom stakeholder from the list
- Better define Kiruna's Area
- Change the plus icon on the map

**Comment by stefffffffffano** [comment]
Good morning! First of all, thanks for your feedback. I have fixed the details you required to be changed for what concerns the map (Kiruna's area and the plus icon) and other details regarding the Popup. My collegue is working on other requirements for the form, he will let you know and close the issue as soon as it is complete.

**Comment by Gianluca280600** [comment]
Thank you for the suggestions. As mentioned by my colleague, I have completed the remaining part of the suggestions you provided. Specifically, I increased the default size of the Description field, reorganized all the fields within the form to avoid empty spaces, and finally enabled the option to remove manually added stakeholders from the list. Thanks again for your feedback. Closing the issue, I wish you good work.
